README.md

```diff
@@ -56,9 +56,9 @@ from the project root. This will create a plugin zip file at
 from the IDE. `<PRODUCT>` can be one of `ijwb, clwb, aswb`.
 
 If the IDE refuses to load the plugin because of version issues, specify the
-correct `ij_product`. These are in the form `<IDE>-oss-<VERSION>` with 
+correct `ij_product`. These are in the form `<IDE>-<VERSION>` with
   * `<IDE>` being one of `intellij-ue, intellij, clion, android-studio`, 
-  * `<VERSION>` being one of `oldest-stable, latest-stable, under-dev`.
+  * `<VERSION>` being one of the aliases `oldest-stable`, `latest-stable`, `under-dev`, `oss-oldest-stable`, `oss-latest-stable`, `oss-under-dev` or direct versions, like `2023.1`, `2023.2`
```

Comment:
The first 2 aliases here should be beta and latest. Please refer to the aliases definition in https://github.com/bazelbuild/intellij/blob/master/intellij_platform_sdk/build_defs.bzl#L4